components/event/sponsors.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="flex items-center mb-2">
+    <p class="text-mtl-infos">Sponsor:</p>
+    <ul>
+      <li v-for="sponsor in sponsors" :key="sponsor._uid" class="px-4 py-2">
+        <a :href="sponsor.link.url" target="_blank">
+          <img :src="sponsor.image" :alt="sponsor.name" class="w-10" />
+        </a>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    /* eslint-disable vue/require-default-prop */
+    sponsors: Array
+  }
```

Comment:
I just realised there is a bunch of `/* eslint-...` disabling in this project :\   We should really strive to avoid that as much as possible as they are simply best practices we ignore without any apparent good reason. 

I'll address the others in another ticket, but while we are here, let's fix this one. 
You can try this is should do the trick: 

```suggestion
  props: {
    sponsors: {
      type: Array,
      required: true
    }
  }
```

Reply:
100% agree! and thank you for this one, I could not see where was the error. FYI, there are still some eslint disable rules (every config files has one) but my knowledge of Nuxt is not strong enough to be sure I can remove them without breaking everything 😛 

addressed in 601a0a2

Reply:
Awesome! And yes for the others I've started a branch here https://github.com/vuemontreal/vuemontreal/pull/150 already :)